pkg/site/connector.go

```diff
@@ -9,11 +9,15 @@ import (
 
 func UpdateBridgeConfigForConnector(siteId string, connector *skupperv1alpha1.Connector, config *qdr.BridgeConfig) {
 	if connector.Spec.Host != "" {
-		UpdateBridgeConfigForConnectorWithHost(siteId, connector, connector.Spec.Host, config)
+		UpdateBridgeConfigForConnectorWithHostProccess(siteId, connector, connector.Spec.Host, "", config)
 	}
 }
 
 func UpdateBridgeConfigForConnectorWithHost(siteId string, connector *skupperv1alpha1.Connector, host string, config *qdr.BridgeConfig) {
+	UpdateBridgeConfigForConnectorWithHostProccess(siteId, connector, host, "", config)
+}
+
+func UpdateBridgeConfigForConnectorWithHostProccess(siteId string, connector *skupperv1alpha1.Connector, host string, processID string, config *qdr.BridgeConfig) {
```

Comment:
Not especially proud of how I went about this - having simply piled on the poor site package's public interface.

Reply:
Nit: typo in function name: Proccess -> Process